lib/reqTemplate.js

```diff
@@ -95,6 +96,48 @@ var globalMethods = {
         };
     },
 
+    /**
+     * Formats a date in a requested format
+     *
+     * @param date date to format
+     * @param [format] optional format, defaults to RFC 822 format
+     */
+    date: function(date, format) {
+
+        function isValidDate(d) {
+            if (Object.prototype.toString.call(d) !== "[object Date]") {
+                return false;
+            }
+            return !isNaN(d.getTime());
+        }
+
+        var origDate = date;
+        format = format || 'rfc822';
+
+        if (typeof date === 'string') {
+            // It's a string, but may be it's just a stringified timestamp.
+            // Check if it actually is.
+            var tempNumDate = parseInt(date);
+            if (!Number.isNaN(tempNumDate)) {
+                date = tempNumDate;
+            }
+        }
+        date = new Date(date);
```

Comment:
This should have been inside the `if` block above, `new Date()` does not seem to accept Date objects.